lib/htmlValidator.js

```diff
@@ -10,6 +10,18 @@ colors.setTheme({
 module.exports = function(app) {
   var params = app.get('params'),
       options,
+      i,
+      errorArr,
```

Comment:
In regards to this [comment](https://github.com/kethinov/roosevelt/pull/97#discussion_r103756750) I was thinking an `h2` tag, but since it's sitting in a `pre` tag. So we just have it in text instead. I'm fine with this given the structure